test/behaviour/resolution/reasoner/variable_role/BUILD

```diff
@@ -0,0 +1,53 @@
+#
+# Copyright (C) 2020 Grakn Labs
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+package(default_visibility = ["//test/behaviour:__subpackages__"])
+load("@graknlabs_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+
+java_test(
+    name = "test",
+    srcs = [
+        "VariableRoleTest.java",
+    ],
+    test_class = "grakn.core.test.behaviour.resolution.reasoner.variable_role.VariableRoleTest",
```

Comment:
is there a useful grouping of "role" tests? instead of `variable_role`?

Reply:
The upcoming RelationInferenceTest (currently WIP in a new PR) will be loosely related; otherwise no.